lib/parallel_tests/test/runner.rb

```diff
@@ -97,7 +97,8 @@ def execute_command(cmd, process_number, num_processes, options)
           # being able to run with for example `-output foo-$TEST_ENV_NUMBER` worked originally and is convenient
           cmd.map! { |c| c.gsub("$TEST_ENV_NUMBER", number).gsub("${TEST_ENV_NUMBER}", number) }
 
-          puts Shellwords.shelljoin(cmd) if report_process_command?(options) && !options[:serialize_stdout]
+          env_str = "TEST_ENV_NUMBER=#{env['TEST_ENV_NUMBER']} PARALLEL_TEST_GROUPS=#{env['PARALLEL_TEST_GROUPS']}"
+          puts "#{env_str} #{Shellwords.shelljoin(cmd)}" if report_process_command?(options) && !options[:serialize_stdout]
```

Comment:
we should reuse this "print command" logic, for example in a `self.printable_command` or so